master/internal/config/config.go

```diff
@@ -261,7 +261,11 @@ func (c *Config) Deprecations() (errs []error) {
 			errs = append(errs, fmt.Errorf(
 				"agent_reattach_enabled is set for resource pool %s but will be ignored; "+
 					"as of 0.21.0 this feature is always on", rp.PoolName,
-			))
+			), fmt.Errorf(
```

Comment:
did we always erroneously give an warning for Kubernetes?

Probably should do a check on resource manager type to make sure we aren't talking about Kubernetes when the user is running agents. Though one general message probably is also fine saying this field is now completely deprecated.

Reply:
@NicholasBlaskey I adjusted the check for the second message, so that it makes sure that the Kubernetes resource manager isn't null. 

Is this also something that needs to be done for the message on line 262?

Reply:
@NicholasBlaskey for the first or second message?